cve_bin_tool/checkers/README.md

```diff
@@ -277,15 +277,67 @@ that include this product. For our example all listings except
 `libexpat, expat` mearly include the target product (`expat` for the
 example SQL query).
 
-### For PyPI packages
+## Helper-Script
+Helper-Script is a tool that takes *packages*(i.e. busybox_1.30.1-4ubuntu9_amd64.deb) as input and returns:
 
-Reading through the above sections before this section is highly recommended.
+> 1. `CONTAINS_PATTERNS` - list of commonly found strings in the binary of the product
+> 2. `FILENAME_PATTERNS` - list of different filename for the product
+> 3. `VERSION_PATTERNS` - list of version patterns found in binary of the product.
+> 4. `VENDOR_PRODUCT` - list of vendor product pairs for the product as they appear in NVD.
 
-1. `CONTAINS_PATTERN` can be any unique string in the whole package and can be found through the same process as in **Choosing contains patterns to detect the library**.
-2. `FILENAME_PATTERNS` should be `[r"METADATA", r"PKG-INFO"]`.
-3. `VERSION_PATTERNS` should be `[r"--generated pattern for cve-bin-tool Name: <package-name> Version: ([0-9]+\.[0-9]+\.[0-9]+)"]`. The strings in `METADATA` and `PKG-INFO` are converted similar to above string for version pattern detection.  
- Note: The version pattern can change depending on the version convention of the package
-5. `VENDOR_PRODUCT` can be found through the same process as in **Finding Vendor Product pairs section**.
+To use this tool, follow the below steps:
+
+***STEP-1*** Download the binary packages for which you wish to create the *Checker* for. A good place to download the packages is from [pkgs.org](https://pkgs.org/search/?q=tomcat).
+
+***STEP-2*** Now that you have your required package, run the tool as a normal python script and then provide the package name as an argument.
+> *NOTE: Do not do `python -m cve-bin-tool/helper_script` as that would import it as a module.*
```

Comment:
Doesn't need to be in this PR, but did you want to add some sort of entry point for your helper script in a future revision?  (e.g. add it to https://github.com/intel/cve-bin-tool/blob/main/setup.py the way we did with csv2cve)